app/views/users/_form.html.erb

```diff
@@ -79,6 +82,10 @@
       <%= render 'ssh_keys/ssh_keys_tab', :f => f, :user => @user %>
     <% end %>
 
+    <% if @editing_self || authorized_for(:controller => 'personal_access_tokens', :action => :view) %>
+      <%= render 'personal_access_tokens/personal_access_tokens_tab', :f => f, :user => @user %>
```

Comment:
would it make sense to manage the entire tab in react?

Reply:
I don't see, why we should. Why could we gain from this?

Reply:
highlight errors?